app/Notifications/ConfirmEmail.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Account;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class ConfirmEmail extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @return array
+     */
+    public function via()
+    {
+        $first = Account::count() == 1;
+        if (! config('monica.signup_double_optin') || $first) {
+            return [];
+        }
+
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+            ->subject(__('confirmation::confirmation.email-title'))
```

Comment:
What is `__('confirmation...` ?

Translations?

Reply:
Yes. This file is a stub copied from the new package.
I guess `__()` is an equivalent of `trans()`. The prefix 'confirmation::' is mandatory, it gets the translation from the package itself.